integreat_cms/cms/templates/organizations/organization_form.html

```diff
@@ -10,28 +10,33 @@
         {% csrf_token %}
         <div class="flex justify-between mb-4">
             <h1 class="heading">
-                {% if form.initial %}
-                    {% with form.name.value as organization_name %}
-                        {% blocktranslate trimmed %}
-                            Edit organization "{{ organization_name }}"
-                        {% endblocktranslate %}
-                    {% endwith %}
+                {% if form.instance.id %}
+                    {% blocktranslate trimmed with organization_name=form.instance.name %}
+                        Edit organization "{{ organization_name }}"
+                    {% endblocktranslate %}
                 {% else %}
                     {% translate "Create new organization" %}
                 {% endif %}
             </h1>
             <div class="flex flex-wrap justify-between gap-4">
                 {% if form.instance.id and perms.cms.delete_organization %}
                     <div class="flex flex-wrap gap-4">
-                        <button title="{% translate "Delete organization" %}"
-                                class="btn confirmation-button btn-red"
-                                data-confirmation-title="{{ delete_dialog_title }}"
-                                data-confirmation-text="{{ delete_dialog_text }}"
-                                data-confirmation-subject="{{ form.name.value }}"
-                                data-action="{% url "delete_organization" region_slug=request.region.slug slug=form.instance.slug %}">
-                            <i icon-name="trash-2" class="mr-2"></i>
-                            {% translate "Delete" %}
-                        </button>
+                        {% if organization.is_used %}
+                            <button title="{{ cannot_delete_title }}" class="btn" disabled>
+                                <i icon-name="trash-2" class="mr-2"></i>
+                                {% translate "Delete" %}
+                            </button>
+                        {% else %}
+                            <button title="{% translate "Delete organization" %}"
+                                    class="btn confirmation-button btn-red"
+                                    data-confirmation-title="{{ delete_dialog_title }}"
+                                    data-confirmation-text="{{ delete_dialog_text }}"
+                                    data-confirmation-subject="{{ form.name.value }}"
+                                    data-action="{% url "delete_organization" region_slug=request.region.slug organization_id=form.instance.id %}">
+                                <i icon-name="trash-2" class="mr-2"></i>
+                                {% translate "Delete" %}
+                            </button>
```

Comment:
This button should not be clickable if the organisation is currently used

On a sidenote: When I clicked the button for testing on an organisation that was used, I got a success message "Organization was successfully deleted", even though it was not actually deleted (because our code in the organisation model made sure that it can only be deleted if it is not used). I think it would be better if we instead showed an error that the organisation could not be deleted.
However, since this should never happen in practice anyways I think it is also fine if you don't implement that :)

Reply:
What would a clean solution for this look like? Raise an exception in the model, catch it in the view and then deciding on this showing an error or a success message?

Reply:
Personally I would just return a boolean from the function, but I think one could also argue for using exceptions. The rest sounds good to me :)

Reply:
Sounds good :)